backend/src/main/java/pl/krkteam/battleships/ships/placing/validation/services/ShipDirectionChecker.java

```diff
@@ -6,7 +6,7 @@
 class ShipDirectionChecker {
     static Direction whatDirection(ShipFromJson shipFromJson) {
         CoordinatesFromJson[] coordinatesFromJsonArray = shipFromJson.getCoordinates();
-        if (coordinatesFromJsonArray.length==1){
+        if (coordinatesFromJsonArray.length == 1) {
```

Comment:
extract expression "coordinatesFromJsonArray.length == 1" to descriptive method

Reply:
doesn't make any sense to extract it